lib/interfaces/chameleon-options.d.ts

```diff
@@ -0,0 +1,31 @@
+import { PlainObjectType } from '../types';
+
+export interface BaseOptions {
+    apply: boolean;
+    name: string;
+}
+
+export interface StrokeWidthOptions extends BaseOptions {
```

Comment:
_**This is just a personal preference of mine:**_

As this interfaces are never implemented by a class they could be just types eg. 

```
export type BaseOptions = {
  apply: boolean;
  name: string;
 }

export type StrokeWidthOptions =  BaseOptions & {
  nonScaling: boolean;
  customVars?: PlainObjectType;
}
```